src/partials/device_selection_information.hbs

```diff
@@ -0,0 +1,28 @@
+{{#each devices as |device|}}
+  {{#ifEq device.id ../selectedDeviceRef.id}}
+    <div class="form-aside">
+      <div class="stack" data-id="stack">
+        <div class="stack stack--small" data-id="stack">
+          <div class="text-block" data-id="textblock">
+            <div class="text-block--overflow-wrap">
+              {{device.type}} sent to:
```

Comment:
This is being rendered for the TOTP case, which isn't quite right as nothing is actually sent. I think it should be the same as the adapter template